classes/PodsAPI.php

```diff
@@ -3640,9 +3640,9 @@ public function save_pod_item( $params ) {
 			// WPML AJAX compatibility
 			if ( is_admin() &&
 				( isset( $_POST['action'] ) && 'wpml_save_job_ajax' === $_POST['action'] ) ||
-				( isset( $_GET['page'] ) && false !== strpos( $_GET['page'], '/menu/languages.php' )
-				 && isset( $_POST['icl_ajx_action'] ) && isset( $_POST['_icl_nonce'] )
-				 && wp_verify_nonce( $_POST['_icl_nonce'], $_POST['icl_ajx_action'] . '_nonce' ) )
```

Comment:
I liked it this way, is the coding standards requiring conditional to be on the left of new line or right side on previous line?

Reply:
Ditto.  The right edge is ragged and may require horizontal scrolling just to see the logic,  To quote the PEAR manual `Keeping the operators at the beginning of the line has two advantages: It is trivial to comment out a particular line during development while keeping syntactically correct code (except of course the first line). Further is the logic kept at the front where it's not forgotten. Scanning such conditions is very easy since they are aligned below each other.`.

Reply:
My apologies - the CS issue here was the single space on front of `&&` for both lines.

Reply:
if you keep the operator at the start which feels a bit more readable the && from line 3641 should move to 42 consistency wise ^^